docs/pages/includes/install-linux.mdx

```diff
@@ -1,220 +1,186 @@
+<Admonition type="tip">
+
+In the example commands below, update `$SYSTEM-ARCH` with the appropriate
```

Comment:
Since the instructions about `$SYSTEM-ARCH` only apply to the "Tarball" tab, I think we can move the first paragraph into that `TabItem` for each edition